packages/date-picker/src/FilterDateRangePicker/components/DateRangeInputField/DateRangeInputField.module.scss

```diff
@@ -43,3 +43,7 @@ $input-disabled-opacity: 0.3;
 .disabled {
   opacity: $input-disabled-opacity;
 }
+
+.fieldMessageIcon > span {
+  align-self: flex-start;
```

Comment:
Is this unique to the DatePicker error messages, or should it be a global change to the FieldMessage?

Reply:
I plan on moving this and the icon update to FieldMessage in a separate PR